overlord/snapstate/snapmgr_test.go

```diff
@@ -96,10 +96,16 @@ func (s *snapmgrTestSuite) TearDownTest(c *C) {
 }
 
 func (s *snapmgrTestSuite) TestStore(c *C) {
```

Comment:
this test needs to do snapstate.ReplaceStore(nil) I think to actually test the cache filling path